pandas/core/series.py

```diff
@@ -2660,16 +2661,6 @@ def sort_values(self, axis=0, ascending=True, inplace=False,
             raise ValueError("This Series is a view of some other array, to "
                              "sort in-place you must create a copy")
 
-        def _try_kind_sort(arr):
```

Comment:
why you changing this?

Reply:
I couldn't get it working to separate the sorting and ascending vs non-ascending ordering further below in the code and changed it to pass it on to `nargsort` that does both at once